src/renderer/components/SyncView/index.js

```diff
@@ -39,8 +40,12 @@ const fileDialogFilters = [
 
 const SyncView = ({ focusState }) => {
   const cx = useStyles()
+  const {
+      metadata: { syncServer }
+  } = useConfig()
   const listenForSyncPeers = focusState === 'focused'
-  const [peers, syncPeer] = usePeers(listenForSyncPeers)
+  const [peers, syncPeer, connectCloud] = usePeers(listenForSyncPeers)
```

Comment:
There are 3 states here:

1. config is loading
2. config is loaded
3. error loading config

Right now it's not clear what happens during the "loading" state (e.g. when syncServer and Encryption key are undefined because the async call has not completed). We should be explicit about the loading state, and render a loading indicator. We should also catch any error with loading the encryption key and metadata. What should we show to the user if there is an error loading metadata or encryption key (which could be due to a future change in the config file format for example, or a corrupt config file).

Reply:
When syncServer is undefined, the button to connect to the cloud is hidden. Does that help?

Does the config loading error handling need to happen for this PR, or is that something that could be handled after? Feels like the code for that already existed.